packages/entity-list/src/components/SearchFilterList/SearchFilterList.js

```diff
@@ -57,15 +56,14 @@ export const StyledSearchFilterButton = styled.div`
 
 const SearchFilterButton = ({setActive, active, label}) =>
   <StyledSearchFilterButton active={active} onClick={() => setActive(!active)}>
-    <StyledSpanSearchFilter>{label}
-      <StyledButton
-        onClick={e => {
-          setActive(false)
-          e.stopPropagation()
-        }}
-        icon="plus"
-        dense />
-    </StyledSpanSearchFilter>
+    <StyledSpanSearchFilter>{label}</StyledSpanSearchFilter>
+    <StyledButton
```

Comment:
When text is cropped, the full text should be visible on mouseover > set title on button